modules/monitoring-supported-remote-write-authentication-settings.adoc

```diff
@@ -0,0 +1,330 @@
+Each sample configures authentication for use with default platform monitoring in the `openshift-monitoring` namespace.
+
+.Sample YAML for AWS Signature Version 4 authentication
+
+The following shows the settings for a `sigv4` secret named `sigv4-credentials` in the `openshift-monitoring` namespace.
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sigv4-credentials
+  namespace: openshift-monitoring
+stringData:
+  access: <AWS_access_key> <1>
+  secret: <AWS_secret_key> <2>
+type: Opaque
+----
+<1> The AWS API access key.
+<2> The AWS API secret key.
+
+The following shows sample AWS Signature Version 4 remote write authentication settings that use a `Secret` object named `sigv4-credentials` in the `openshift-monitoring` namespace:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://authorization.example.com/api/write"
+        sigv4:
+          region: <AWS_region> <1>
+          accessKey:
+            name: sigv4-credentials <2>
+            key: access
+          secretKey:
+            name: sigv4-credentials <2>
+            key: secret
+          profile: <AWS_profile_name> <3>
+          roleArn: <AWS_role_arn> <4>
+----
+<1> The AWS region.
+<2> The name of the `Secret` object containing the AWS API access credentials.
+<3> The name of the AWS profile that is being used to authenticate.
+<4> The unique identifier for the Amazon Resource Name (ARN) assigned to your role.
+
+
+.Sample YAML for basic authentication
+
+The following shows sample basic authentication settings for a `Secret` object named `rw-basic-auth` in the `openshift-monitoring` namespace: 
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: rw-basic-auth
+  namespace: openshift-monitoring
+stringData:
+  user: <basic_username> <1>
+  password: <basic_password> <2>
+type: Opaque
+----
+<1> The username.
+<2> The password.
+
+The following sample shows a `basicAuth` remote write configuration that uses a `Secret` object named `rw-basic-auth` in the `openshift-monitoring` namespace.
+It assumes that you have already set up authentication credentials for the endpoint. 
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+      remoteWrite:
+      - url: "https://basicauth.example.com/api/write"
+        basicAuth:
+          username:
+            name: rw-basic-auth <1>
+            key: user
+          password:
+            name: rw-basic-auth <1>
+            key: password
+----
+<1> The name of the `Secret` object that contains the authentication credentials.
+
+.Sample YAML for bearer token authentication using a token file
```

Comment:
as I commented above, I believe that we only need an example for `bearer`.

Reply:
deleted